ml_metadata/tools/mlmd_bench/thread_runner.cc

```diff
@@ -86,15 +87,16 @@ tensorflow::Status ExecuteWorkload(const int64 work_items_start_index,
 }
 
 // Merges all the thread stats inside `thread_stats_list` into a workload stats
-// and reports the workload's performance according to that.
+// and reports the workload's performance according to that. Also, store the
+// performance inside `workload_summary`.
 void MergeThreadStatsAndReport(const std::string workload_name,
-                               ThreadStats thread_stats_list[], int64 size) {
+                               ThreadStats thread_stats_list[], int64 size,
```

Comment:
for the array var, please see the part 2 cl that used a vector<>, probably merge the upstream once that cl is submitted.

Reply:
Got it. Will sync with head once internal CL has been published.